guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,7 +1,11 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
-When you run a remote job on hosts, for every host, {Project} performs the following actions to find a remote execution {SmartProxy} to use.
+{Project} prefers remote execution on host's registered_through proxy.
+You must *enable remote_execution_prefer_registered_through_proxy* manually from *Administer > Settings > Content*.
+This will allow selecting a {SmartProxies} to perform the REX job on the hosts to which they are registered to.
```

Comment:
```suggestion
This will select the {SmartProxy} to which the hosts are registered, to perform the REX job.
```

Reply:
@mjivraja I think this sounds better. What do you think @maximiliankolb ?

Reply:
Thanks @AkshayGadhaveRH 
Wait for Maxi if he has something for us.

Reply:
I think the whole first paragraph should be reworded.

a) describe what REX is
b) how SmartProxies play their part to run jobs on managed hosts.

Assuming a host is always only registered to one smartproxy, either Foreman or a Smart Proxy, then describing that you _can_ use the "registered through" Smart Proxy sounds weird to me.

Reply:
Addressed. :+1: 
Kept `SmartProxies`